Gaijinizer patch/Map536.txt

```diff
@@ -4,1053 +4,1342 @@
+> CONTEXT: Map536/events/7/pages/0/609/Dialogue/enemy29
+How about you stop hiding and come out?
 
+You're about to return to the other side anyway.#Ron
 > END STRING
 
 > BEGIN STRING
 ………………………
 
 ……いつから気付いてたの？
-> CONTEXT: Map536/events/7/pages/0/624/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map536/events/7/pages/0/624/Dialogue/sirousagi_fc1
+.........
 
+How long did you know I was there?#Ron
 > END STRING
 
 > BEGIN STRING
 \\C[2]ずーーーーっっっと最初っから♪
 
 キミがボクに『手紙』を渡したときからだよ。
-> CONTEXT: Map536/events/7/pages/0/628/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map536/events/7/pages/0/628/Dialogue/enemy29
+\\C[2]I knew the whooooooole time... ♪
 
+Ever since you handed me that "letter"...#Ron
 > END STRING
 
 > BEGIN STRING
 おおかた、キミは『白兎』を模倣しきれなかったんだろう？
 
 だから、ボクに『手紙』なんて渡してきた……
-> CONTEXT: Map536/events/7/pages/0/632/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/632/Dialogue/enemy29
+It was probably impossible for you to emulate
+White Rabbit. That's why you gave me that
+letter.#RonIffy
 > END STRING
 
 > BEGIN STRING
 \\C[2]白兎は『手紙』なんて寄越さないんだよ……
 
 \\C[0]ボクと彼……いや今は彼女かな？
 とにかくボクたちは、\\C[2]同じ空間での情報共有はしない。
-> CONTEXT: Map536/events/7/pages/0/636/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map536/events/7/pages/0/636/Dialogue/enemy29
+\\C[2]White Rabbit wouldn't ever hand over a letter.\\C[0]
 
+Me and him, or "her" now... Either way we don't
+\\C[2]share information in the same space\\C[0].#RonIffy
 > END STRING
 
 > BEGIN STRING
 ………どういうことだい？
-> CONTEXT: Map536/events/7/pages/0/642/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/642/Dialogue/hyakki
+What are you talking about?#Ron
 > END STRING
 
 > BEGIN STRING
 ボクたちは本来、形を持たないんだよ……
 
 方程式とか、定理とかと同じ……
 その世界において、必要であり、必要でないもの……
-> CONTEXT: Map536/events/7/pages/0/644/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/644/Dialogue/enemy29
+We are essentially formless. Like equations and
+theorems, we are both necessary and unnecessary
+in the world.#DL
 > END STRING
 
 > BEGIN STRING
 今のボクは\\C[6]『上位存在』\\C[0]によって目的を与えられ
 色々な権限を与えられて、実体を持つことも許されている…
 
 でも本来は、同一空間で混ざってはいけないものなんだ。
-> CONTEXT: Map536/events/7/pages/0/649/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/649/Dialogue/enemy29
+The form I'm in right now was permitted to
+materialize by the \\C[6]higher powers \\C[0]with an objective
+to complete, and with various authorities. But
+it cannot be allowed to mix in the exact same
+space.#RonIffy
 > END STRING
 
 > BEGIN STRING
 極端な話をすると……
 
 『1＋1＝2』と『1＋1＝田』の"公式"が同時に存在したら
 \\C[2]世界の法則が連鎖的にぐちゃぐちゃになってしまう…
-> CONTEXT: Map536/events/7/pages/0/654/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map536/events/7/pages/0/654/Dialogue/enemy29
+To give an extreme example, if the equations
+"1 + 1 = 2", and "1 + 1 = 3" both officially
+existed, \\C[2]it would cause a chain reaction of damage
+to the world's natural laws\\C[0].#RonIffyWording
 > END STRING
```

Comment:
Minor changes to flow:
```
To give an extreme example, if the equations
"1 + 1 = 2", and "1 + 1 = 3" both officially
existed, \\C[2]it would cause a chain reaction that
would heavily damage the world's natural laws\\C[0].
```